package pa.chan.main_page.data.userException

object ConnectionException : Throwable(), UserException {
}